DigitalLibraryofTennessee/check_and_harvest
# Address DLTN_XSLT-223.

## What does this do

This modifies check and harvest to be able to catch bad thumbnails from CMHF hatch.

## More Details

The previous version expected URLs to be formatted according to how they are in production.  Unfortunately, the OAI-PMH response has a different URL version that redirects to the new version.  Because of this, we weren't properly testing thumbnails and some records were still getting through.

This:

* Fixes the issue with restricted items from hatch3
* Positions this test to be used for other thumbnail scenarios.
* Adds test data that is similar to what we'd find in OAI-PMH responses.
* Adds a checksum for the thumbnail that is used for items in TSLA maps but not yet published so we can test those if needed.

## How to test

* Does Travis pass?
* If you want to go nuts, pull in hatch and see if any items you don't expect get through (not necessary since I'm also doing that and it will take a long time).

## Interested Parties

@mlhale7
